lib/helpers.coffee

```diff
@@ -65,9 +66,11 @@ module.exports = Helpers =
     if lineNumber > maxLine
       throw new Error("Line number (#{lineNumber}) greater than maximum line (#{maxLine})")
     if typeof colStart isnt 'number' or colStart < 0
-      colStart = (textEditor.indentationForBufferRow(lineNumber) * textEditor.getTabLength())
-      if colStart isnt 0
-        colStart -= 1
+      indentation = buffer.lineForRow(lineNumber).match(/^\s+/)
+      if indentation? and indentation.length = 1
```

Comment:
`indentation?` <-- you dont need the `?` here because if it's null, it'll be falsy already